wgpu-hal/src/gles/adapter.rs

```diff
@@ -208,11 +208,9 @@ impl super::Adapter {
 
         let min_uniform_buffer_offset_alignment =
             gl.get_parameter_i32(glow::UNIFORM_BUFFER_OFFSET_ALIGNMENT);
-        let min_storage_buffer_offset_alignment = if cfg!(target_arch = "wasm32") {
-            256
-        } else {
-            gl.get_parameter_i32(glow::SHADER_STORAGE_BUFFER_OFFSET_ALIGNMENT)
-        };
+        let min_storage_buffer_offset_alignment = gl
+            .get_parameter_i32(glow::SHADER_STORAGE_BUFFER_OFFSET_ALIGNMENT)
+            .max(256);
```

Comment:
We shouldn't hard-code the default here without referencing to it directly. I think storage buffers can only be queried on GLES 3.1, so here we just need to check for the version, and fall back to some trivial default instead of the actual limits default.
```rust
let min_storage_buffer_offset_alignment = if ver >= (3, 1) {
  gl.get_parameter_i32(glow::SHADER_STORAGE_BUFFER_OFFSET_ALIGNMENT)
} else {
  1
};
```